docs/tutorials/timeseries/forecasting-quickstart.md

```diff
@@ -9,6 +9,20 @@ Via a simple `fit()` call, AutoGluon can train
 
 to produce multi-step ahead _probabilistic_ forecasts for univariate time series data. 
 
+---
+**NOTE**
+
+`autogluon.timeseries` depends on Apache MXNet. Please install MXNet by
+
+```shell
+python -m pip install mxnet>=1.9
+```
+
+or, if you are using a GPU with a matching MXNet package for your CUDA driver. See the 
+MXNet [documentation](https://mxnet.apache.org/versions/1.9.1/get_started?) for more info.
+
+---
+
```

Comment:
General comment: We may want to figure out if we can make nicer Notice boxes in tutorials, similar to what we can do in .rst files like here with `Warning` and `Note`: http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1888/3/install.html